javasource/myfirstmodule/interfaces/CoreProxy.java

```diff
@@ -59,6 +60,11 @@ public default void storeFileDocumentContent(IContext ic, IMendixObject imo, Str
     public default MicroflowCallBuilder microflowCall(String microflowName) {
         return Core.microflowCall(microflowName);
     }
+
+    public default IMendixObject retrieveId(IContext ic, IMendixIdentifier imi) throws CoreException {
```

Comment:
Can you change the parameters to: `(IContext context, IMendixIdentifier id)` to reflect: https://apidocs.rnd.mendix.com/8/runtime/com/mendix/core/Core.html#retrieveId(com.mendix.systemwideinterfaces.core.IContext,com.mendix.systemwideinterfaces.core.IMendixIdentifier)